.docker/nginx/browser.conf

```diff
@@ -1,3 +1,10 @@
+resolver 127.0.0.11;
+
+map $cookie_XDEBUG_SESSION $fastcgi_pass {
```

Comment:
if this needs to set something specific in the IDE/host, it would need documentation

Reply:
Should work with any value.